### PR TITLE
Rotate refresh icon

### PR DESCRIPTION
**Pull request by anteriovieira**
Maybe it is not the best way to implement, but it would be nice to have a behavior that indicase the refresh.

refresh-icon

### CODE REVIEW

File: src/devtools/App.vue [hunk 1]
```diff
@@ -78,7 +79,13 @@ export default {
       }
     },
     refresh () {
+      this.refreshRotate += 180
+      const refleshIcon = this.$el.querySelector('.refresh-icon')
```
Comment (posva) [inline src/devtools/App.vue]: you can use a ref on the element instead

File: src/devtools/App.vue [hunk 2]
```diff
@@ -78,7 +79,13 @@ export default {
       }
     },
     refresh () {
+      let refreshIcon = this.$refs.refresh
+      this.refreshRotate += 180
+
       bridge.send('refresh')
+      bridge.on('flush', () => {
```
Comment (posva) [inline src/devtools/App.vue]: I forgot to say that the bridge.on should be setup only once

**Comment by anteriovieira** [comment]
Hi @posva , what do you think about changing the icon? An idea would be to leave the same chrome refresh icon.

<p align="center">

</p>

File: src/devtools/global.styl [hunk 3]
```diff
@@ -95,3 +95,17 @@ $arrow-color = #444
     text-align center
     padding 0.5em
     margin 0 auto
+
+@-moz-keyframes rotate
+  0%
+    -moz-transform rotate(0deg)
```
Comment (posva) [inline src/devtools/global.styl]: No need for prefixes because stylus will add them. So keeping the `@keyframes` rule with only `transform` is enough

**Comment by anteriovieira** [comment]
@posva , what do you think of these latest changes?

**Pull request merged**